src/components/Friends.js

```diff
@@ -6,35 +6,53 @@ class Friends extends React.Component {
     constructor() {
         super();
         this.state = {
-            friends: []
+            friends: [],
+            activeFriend: []
         }
     }
 
     componentDidMount() {
+
         axios.get('http://localhost:5000/friends')
-        .then(res => this.setState({friends: res.data}) )
+        .then(res => {console.log(res); this.setState({friends: res.data}) })
+        .catch(err => err);
       }
+    
 
```

Comment:
Line 18 - Why are you trying to return `err` in your `.catch` callback? Did you originally intend to log this value in the console (as with your other `axios` calls)?